src/sdg/core/apps.py

```diff
@@ -37,3 +37,6 @@ class CoreConfig(AppConfig):
     def ready(self):
         post_migrate.connect(register_project_models, sender=self)
         post_migrate.connect(load_app_fixtures, sender=self)
+
+        # This makes the tasks available in the Django Admin.
+        from . import tasks
```

Comment:
Can you clarify why this is needed? The comment may not be specific, as it already shows in the admin without it.

Reply:
Ah, I was looking for that import but could not find it.